app/models/role.rb

```diff
@@ -0,0 +1,13 @@
+# == Schema Information
```

Comment:
I think you don't need this table. For this reason please use 'enum' functionality in users model.